README.md

```diff
@@ -9,7 +9,7 @@ One option is to use the Docker all-in-one launch as described in the [OpenShift
 
 ### The project ###
 
-If you don't have a project setup all ready, go ahead and take care of that
+After logging in (`oc login`, when any ID/Password combination should be fine), if you don't have a project setup all ready, go ahead and take care of that
```

Comment:
s/when/with/ but more importantly any combination is only fine for the dev openshift environments we run, the one they're using will hopefully be more locked down.  so i'd just say to login with oc login.